packages/core/src/browser/core-preferences.ts

```diff
@@ -57,6 +57,12 @@ export const corePreferenceSchema: PreferenceSchema = {
         'workbench.iconTheme': {
             type: ['string', 'null'],
             description: "Specifies the icon theme used in the workbench or 'null' to not show any file icons."
+        },
+        'workbench.tree.renderIndentGuides': {
```

Comment:
Is this preference definition aligned with VS Code?

Reply:
Yes.